### PR TITLE
Fetch mto entitlement info for prime

### PR DESCRIPTION
**Pull request by ralren**
## Description

Updated the Entitlement endpoint to no longer return static data. Copied for the Prime API.

## Setup

Play with the two endpoints in `milmovelocal:3000/swagger-ui/ghc.html`

```sh
make server_run
make client_run
```

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/169155622) for this change

### CODE REVIEW

File: pkg/assets/assets.go [hunk 1]
```diff
@@ -453,12 +453,9 @@ func AssetNames() []string {
 
 // _bindata is a table, holding each asset generator, mapped to its name.
 var _bindata = map[string]func() (*asset, error){
-	"pkg/paperwork/formtemplates/form1203template.png": pkgPaperworkFormtemplatesForm1203templatePng,
-
+	"pkg/paperwork/formtemplates/form1203template.png":                 pkgPaperworkFormtemplatesForm1203templatePng,
```
Comment (mkrump) [inline pkg/assets/assets.go]: This file keeps causing diffs for others on master, so just double check this is a real change https://ustcdp3.slack.com/archives/CP6PTUPQF/p1572534533003500.

File: pkg/handlers/primeapi/move_task_order.go [hunk 2]
```diff
@@ -19,10 +19,12 @@ import (
 	"github.com/transcom/mymove/pkg/handlers"
 )
```
Comment (mkrump) [inline pkg/handlers/primeapi/move_task_order.go]: 👍 on filling these in

File: swagger/prime.yaml [hunk 3]
```diff
@@ -104,7 +104,100 @@ paths:
...
+    get:
+      produces:
+        - application/json
+      parameters: []
+      tags:
+        - moveTaskOrder
+        - prime
+      responses:
+        '200':
+          description: Successfully retrieved move task order
```
Comment (mkrump) [inline swagger/prime.yaml]: Successfully retrieved entitlement

**Comment by ralren** [comment]
@mkrump should be good to be looked at again.

**Comment by mkrump** [comment]
looks good. thanks for the updates!

**Pull request merged**